<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
<suite thread-count="-1" guice-stage="DEVELOPMENT" name="Failed suite [Suite]">
  <listeners>
    <listener class-name="com.dunkin.shadman.onlineStore.helper.listener.RetryListener"/>
  </listeners>
  <test name="Test(failed)">
    <classes>
      <class name="com.dunkin.shadman.onlineStore.testScripts.Test1">
        <methods>
          <include name="beforeClass"/>
          <include name="loginTest1"/>
          <include name="beforeMethod"/>
          <include name="beforeSuite"/>
          <include name="afterMethod"/>
          <include name="loginTest3"/>
        </methods>
      </class> <!-- com.dunkin.shadman.onlineStore.testScripts.Test1 -->
      <class name="com.dunkin.shadman.onlineStore.testScripts.Test2">
        <methods>
          <include name="loginTest1"/>
          <include name="beforeClass"/>
          <include name="beforeMethod"/>
          <include name="beforeSuite"/>
          <include name="afterMethod"/>
          <include name="loginTest3"/>
        </methods>
      </class> <!-- com.dunkin.shadman.onlineStore.testScripts.Test2 -->
    </classes>
  </test> <!-- Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
